### DISCUSSION TITLE
Getting error as 'An item with the same key has already been added. Key: ARIALN.TTF' when try to read dwg file.

### DISCUSSION THREAD

**Discussion by rohan515** [DomCR/ACadSharp#318] [General]
Getting error  when trying to read dwg file, file link https://github.com/rohan515/Cad-files/blob/main/1.dwg
![dwg1](https://github.com/DomCR/ACadSharp/assets/17959422/023382c5-812b-4af9-9c7c-07b88c83a5d2)

**Comment by DomCR** [comment]
The user already created an issue:

- https://github.com/DomCR/ACadSharp/issues/319
